pkg/clusterregistryconfig/flags.go

```diff
@@ -0,0 +1,369 @@
+package clusterregistryconfig
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/openshift/rosa/pkg/helper"
+	"github.com/openshift/rosa/pkg/input"
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/ocm"
+)
+
+const (
+	allowedRegistriesFlag          = "allowed-registries"
+	insecureRegistriesFlag         = "insecure-registries"
+	blockedRegistriesFlag          = "blocked-registries"
+	platformAllowlistFlag          = "platform-allowlist"
+	additionalTrustedCaFlag        = "registry-additional-trusted-ca"
```

Comment:
Please mention this should be a file path on the flag

Reply:
Added file path